pkg/functions/runner.go

```diff
@@ -218,8 +218,9 @@ func runPython(ctx context.Context, job *Job) (err error) {
 	}
 
 	// Run
+	listenAddress := net.JoinHostPort(job.Host, job.Port)
 	if job.verbose {
-		fmt.Printf("PORT=%v ./.venv/bin/python ./service/main.py\n", job.Port)
+		fmt.Printf("PORT=%v LISTEN_ADDRESS=%v ./.venv/bin/python ./service/main.py\n", job.Port, listenAddress)
```

Comment:
Will ./.venv/bin/python do all the job? Don't we need ./.venv/bin/activate?

Reply:
.venv/bin/python is merely a symlink to /usr/bin/python.
I guess python must have some logic checking arg[0].